module/metrics/transaction_validation.go

```diff
@@ -0,0 +1,52 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/onflow/flow-go/module"
+)
+
+type NamespaceType string
+
+const (
+	AccessNamespace     NamespaceType = namespaceAccess
+	CollectionNamespace NamespaceType = namespaceCollection
+)
+
+// TransactionValidationCollector the metrics for transaction validation functionality
+type TransactionValidationCollector struct {
+	transactionValidated        prometheus.Counter
+	transactionValidationFailed *prometheus.CounterVec
+}
+
+// interface check
+var _ module.TransactionValidationMetrics = (*TransactionValidationCollector)(nil)
+
+// NewTransactionValidationCollector creates new instance of TransactionValidationCollector
+func NewTransactionValidationCollector(namespace NamespaceType) *TransactionValidationCollector {
```

Comment:
Let's use the same namespace here regardless of the node being constructed (`access` is fine). The same metric data is being produced, regardless of whether transaction validation is running on a Collection or Consensus node, so we should use the same qualified metric name.

If we want to see transaction validation metrics from a specific node role, labels are the best tool to do that, and we already have `role` labels defined for all our metrics that allow us to do this. 

Namespaces help to produce consistently structured fully qualified metric names, but they shouldn't strictly correspond to node roles if a metric is used across multiple roles. For example, the finalized height metric uses the namespace `consensus`, but all node roles produce this metric (with the same qualified name), because all nodes follow consensus.